packages/core/contracts/optimistic-assertor/implementation/OptimisticAssertor.sol

```diff
@@ -167,11 +169,10 @@ contract OptimisticAssertor is Lockable, OptimisticAssertorInterface, Ownable {
             emit AssertionSettled(assertionId, assertion.proposer, false, true);
         } else {
             // Dispute, settle with the disputer
-            int256 dvmResolvedPrice =
+            int256 resolvedPrice =
                 _getOracle(assertionId).getPrice(identifier, assertion.assertionTime, _stampAssertion(assertionId)); // Revert if price not resolved.
 
-            assertion.settlementResolution = dvmResolvedPrice == 1e18;
-            // todo: if (assertion.useDisputeResolution)
+            assertion.settlementResolution = resolvedPrice == 1e18;
```

Comment:
I think it would be better to store the right value always with something like:
`assertion.settlementResolution = assertion.useDisputeResolution? resolvedPrice == 1e18: false;` otherwise the value is not always true and it could be accesible with the implicit getter of the public variable `assertions` right?

Reply:
fixed in f0b58ae6bb1dfa521e11a06991cd6190685c548c